extensions/agent/client/src/dialogs/pickScheduleDialog.ts

```diff
@@ -0,0 +1,94 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+ 'use strict';
+import * as sqlops from 'sqlops';
+import * as vscode from 'vscode';
+import { PickScheduleData } from '../data/pickScheduleData';
+
+export class PickScheduleDialog {
+
+	// TODO: localize
+	// Top level
+	private readonly DialogTitle: string = 'Job Schedules';
+	private readonly OkButtonText: string = 'OK';
+	private readonly CancelButtonText: string = 'Cancel';
+	private readonly SchedulesTabText: string = 'Schedules';
+
+	// UI Components
+	private dialog: sqlops.window.modelviewdialog.Dialog;
+	private scheduleTab: sqlops.window.modelviewdialog.DialogTab;
+	private schedulesTable: sqlops.TableComponent;s
```

Comment:
extra trailing letter

Reply:
not sure why this doesn't break the build?

Reply:
funny.  this syntax declares a variable name `s`.